src/test/java/edu/kis/vh/nursery/RhymersDemo.java

```diff
@@ -1,35 +1,42 @@
 package edu.kis.vh.nursery;
 
-import edu.kis.vh.nursery.defaultCountingOutRhymer;
-import edu.kis.vh.nursery.HanoiRhymer;
 import edu.kis.vh.nursery.factory.DefaultRhymersFactory;
-import edu.kis.vh.nursery.factory.Rhymersfactory;
+import edu.kis.vh.nursery.factory.RhymersFactory;
 
 class RhymersDemo {
 
-	public static void main(String[] args) {
-		Rhymersfactory factory = new DefaultRhymersFactory();
-		
-		defaultCountingOutRhymer[] rhymers = { factory.GetStandardRhymer(), factory.GetFalseRhymer(),
-				factory.GetFIFORhymer(), factory.GetHanoiRhymer()};
-		
-		for (int i = 1; i < 15; i++)
-			for (int j = 0; j < 3; j++)
-				rhymers[j].countIn(i);
-		
-		java.util.Random rn = new java.util.Random();
-		for (int i = 1; i < 15; i++)
-			rhymers[3].countIn(rn.nextInt(20));
-		
-		for (int i = 0; i < rhymers.length; i++) {
-			while (!rhymers[i].callCheck())
-				System.out.print(rhymers[i].countOut() + "  ");
-			System.out.println();
-		}
-		
-		System.out.println("total rejected is "
-				+ ((HanoiRhymer) rhymers[3]).reportRejected());
-		
-	}
-	
+    private static final int LOOPFORTO15 = 15;
```

Comment:
Most of the tasks did not concern src/test classes. 
A constant value should not be a part of the name (if we would like to change loop iterations, why should we change the constant name).